code/modules/mod/modules/modules_general.dm

```diff
@@ -0,0 +1,436 @@
+//General modules for MODsuits
+
+///Storage - Adds a storage component to the suit.
+/obj/item/mod/module/storage
+	name = "MOD storage module"
+	desc = "What amounts to a series of integrated storage compartments and specialized pockets installed across \
+		the surface of the suit, useful for storing various bits, and or bobs."
+	icon_state = "storage"
+	complexity = 3
+	incompatible_modules = list(/obj/item/mod/module/storage, /obj/item/mod/module/plate_compression)
+	/// Max weight class of items in the storage.
+	var/max_w_class = WEIGHT_CLASS_NORMAL
+	/// Max combined weight of all items in the storage.
+	var/max_combined_w_class = 15
+	/// Max amount of items in the storage.
+	var/max_items = 7
+	var/obj/item/storage/backpack/modstorage/bag
```

Comment:
Clear this in destroy.

Reply:
this is done as parent destroy calls `on_uninstall`